src/Workspace/index.jsx

```diff
@@ -734,19 +735,7 @@ class Workspace extends React.Component {
         </div>
       );
     } if (this.state.defaultWorkspace && this.state.connectedStatus) {
-      // If this commons does not use Hatchery to spawn workspaces, then this
-      // default workspace is shown.
-      return (
-        <div className='workspace__default'>
-          <iframe
-            title='Workspace'
-            frameBorder='0'
-            className='workspace__iframe'
-            src={workspaceUrl}
-            onLoad={this.oniframeLoad}
-          />
-        </div>
-      );
+      return <Redirect to={workspaceUrl} />;
```

Comment:
why redirect use to `/lw-workspace` in this case? Shouldn't just redirect user to the workspace error page?